pkg/operator/ceph/cluster/mgr/dashboard.go

```diff
@@ -197,30 +200,70 @@ func (c *Cluster) createSelfSignedCert() (bool, error) {
 	return false, nil
 }
 
+// CephVersionsNewCephDashboardCommand check if Ceph versions have the latest Ceph dashboard command
+func CephVersionsNewCephDashboardCommand(c *client.ClusterInfo) bool {
+	if (c.CephVersion.IsNautilus() && c.CephVersion.IsAtLeast(cephver.CephVersion{Major: 14, Minor: 2, Extra: 17})) ||
+		(c.CephVersion.IsOctopus() && c.CephVersion.IsAtLeast(cephver.CephVersion{Major: 15, Minor: 2, Extra: 10})) ||
+		c.CephVersion.IsAtLeastPacific() || c.CephVersion.IsAtLeastQuincy() {
```

Comment:
`IsAtLeastPacific()` will already cover Quincy and all future versions
```suggestion
		c.CephVersion.IsAtLeastPacific() {
```

Reply:
Done.